storage/remote/write_handler.go

```diff
@@ -97,8 +97,8 @@ func (h *writeHandler) write(ctx context.Context, req *prompb.WriteRequest) (err
 		for _, s := range ts.Samples {
 			_, err = app.Append(0, labels, s.Timestamp, s.Value)
 			if err != nil {
```

Comment:
Can we unwrap once for all here? switch unwrapErr := errors.Unwrap

Reply:
Ok, done !